apps/deploy/upload-dapp-chain/src/common/hellpers/get-list-repo.ts

```diff
@@ -0,0 +1,13 @@
+import { readFileSync } from "fs";
+import { load } from "js-yaml";
+
+// eslint-disable-next-line consistent-return
+export function getListRepo(): { repo: string }[] {
+  const path = "/../../../src/common/repos/dapps-repo.yaml";
```

Comment:
I'm not sure that it is good way to keep it in `src` maybe it will be better to take path from process.env and keep it somewhere on server.

Reply:
moved to `.env`